recipes/example/meta.yaml

```diff
@@ -17,66 +15,55 @@ source:
   # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
   # and otherwise fall back to archive: 
   # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
-  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+  sha256: 7800f0ba704bfbfa0564538c790a3551142bb5e23cbe7282e69d4f9ffe0d9536
   # sha256 is the preferred checksum -- you can get it for a file with:
   #  `openssl sha256 <file name>`.
   # You may need the openssl package, available on conda-forge:
   #  `conda install openssl -c conda-forge``
 
 build:
-  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
-  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
-  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
-  # noarch: python
+  noarch: python
   number: 0
-  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
-  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
-  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   build:
-    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
-    # Compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
   host:
     - python
     - pip
   run:
-    - python
+    - matplotlib>=3.0.0
+    - numpy>=1.12.0
+    - pyqtgraph>=0.10.0
+    - h5py>=2.8.0
+    - lmfit>=1.0
+
 
 test:
   # Some packages might need a `test/commands` key to check CLI.
   # List all the packages/modules that `run_test.py` imports.
   imports:
-    - simplejson
-    - simplejson.tests
+    - qcodes
+    - pytest
+    - pytest-qt
+    - mypy
+    - PyQt5-stubs
+    - git+https://github.com/numpy/numpy-stubs.git
```

Comment:
I think this should just be `- plottr` which will basically test that `plottr` can be opened when the conda package is installed